core/core_integrations/slack.js

```diff
@@ -605,7 +585,7 @@ exports.stop = function() {
 			while (numSocketsToShutDown > 0 && !clearedMessageQueue && !clearedTimeouts) {
 				sleep.usleep(500000);
 			}
-			callback(null, "finished");
+			callback(null, 'finished');
 		}
 	], function(err, results) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['results' is defined but never used.](https://www.codacy.com/app/theprojectshop/Kassy/file/2549394133/issues/source?bid=3282987&fileBranchId=3288593#l590)